src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -166,12 +166,9 @@ public PowerShellContext(ILogger logger, bool isPSReadLineEnabled)
         /// <returns></returns>
         public static Runspace CreateRunspace(PSHost psHost)
         {
-            var initialSessionState = InitialSessionState.CreateDefault2();
+            var initialSessionState = InitialSessionState.CreateDefault();
```

Comment:
I think I changed this over naively when I started getting `null`s from the tests (not realising what was going on). So this PR should revert it. But I recall @SteveL-MSFT mentioning that `CreateDefault()` turns out to perform better despite the contrary intention...

Reply:
For reference PowerShell/PowerShell#5104

Reply:
The problem is that CreateDefault loads the standard modules as snap-ins so they wouldn't show up in `Get-Module` I don't think this is a desired behavior and think it should be CreateDefault2 and add set the psmodulepath so that they can be autoloaded

Reply:
@tylerl0706 If that's the default, wouldn't that be desired behavior?

Reply:
Well it's different than `pwsh.exe` or `powershell.exe` which, at this time, shows the standard modules in `Get-Module` - so those (in pwsh.exe) are brought in as modules not snap-ins.

I think we want to be as close to what `pwsh.exe` is doing as possible.

Reply:
Funny story, I changed this when the SDK wasn't binding properly because I thought it might be involved and then forgot about it.

Just changed it back, and PowerShell 6.1 tests error out massively because of the Windows PowerShell module path (I think...).

Anyway, we have to keep `CreateDefault2()` for the tests to pass.